dw_client/watchtower.drush.inc

```diff
@@ -175,6 +175,11 @@ function watchtower_get_default_report() {
     $metrics['site:patches'] = $patches;
   }
 
+  // Add Hacked.
+  if ($hacked = watchtower_check_hacked($modules)) {
```

Comment:
watchtower_check_hacked() always returns an array no ? if so, not sure that "if" is needed. we could return the hacked array only if the count($check_changed) is > 0 and returns false otherwhise and then that if would be needed.